stories/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,385 @@
+          sunt aliqua put a bird on it squid single-origin coffee nulla
+          assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer
+          labore wes anderson cred nesciunt sapiente ea proident.
+        </AccordionBody>
+      </AccordionItem>
+    </Accordion>
+  );
+};
+
+GruppiDiElementiRichiudibili.storyName = 'Gruppi di elementi richiudibili';
+
+export const _Accordion = () => {
+  const [collapseElementOpen, setCollapseElement] = useState('');
+  return (
+    <Accordion>
+      <AccordionItem>
+        <AccordionHeader
+          active={collapseElementOpen === '1'}
+          onToggle={() =>
+            setCollapseElement(collapseElementOpen !== '1' ? '1' : '')
+          }
+        >
+          Accordion Group Item #1
+        </AccordionHeader>
+        <AccordionBody active={collapseElementOpen === '1'}>
+          Vestibulum hendrerit ultrices nibh, sed pharetra lacus ultrices eget.
+          Morbi et ipsum et sapien dapibus facilisis. Integer eget semper nibh.
+          Proin enim nulla, egestas ac rutrum eget, ullamcorper nec turpis.
+        </AccordionBody>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader
+          active={collapseElementOpen === '2'}
+          onToggle={() =>
+            setCollapseElement(collapseElementOpen !== '2' ? '2' : '')
+          }
+        >
+          Accordion Group Item #2
+        </AccordionHeader>
+        <AccordionBody active={collapseElementOpen === '2'}>
+          Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer
+          farm-to-table, raw denim aesthetic synth nesciunt you probably
+          haven&apos;t heard of them accusamus labore sustainable VHS.
+        </AccordionBody>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader
+          active={collapseElementOpen === '3'}
+          onToggle={() =>
+            setCollapseElement(collapseElementOpen !== '3' ? '3' : '')
+          }
+        >
+          Accordion Group Item #3
+        </AccordionHeader>
+        <AccordionBody active={collapseElementOpen === '3'}>
+          Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor,
+          sunt aliqua put a bird on it squid single-origin coffee nulla
+          assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer
+          labore wes anderson cred nesciunt sapiente ea proident.
+        </AccordionBody>
+      </AccordionItem>
+    </Accordion>
+  );
+};
+
+export const AccordionAnnidati = () => {
+  const [collapseOpen1, setCollapseOpen1] = useState(true);
+  const [collapseOpen2, setCollapseOpen2] = useState(false);
+  const [collapseOpen3, setCollapseOpen3] = useState(false);
+  const [nestedCollapseOpen1, setNestedCollapseOpen1] = useState(true);
+  const [nestedCollapseOpen2, setNestedCollapseOpen2] = useState(false);
+  const [nestedCollapseOpen3, setNestedCollapseOpen3] = useState(false);
```

Comment:
nested collapsed should be mutually managed, see https://italia.github.io/bootstrap-italia/docs/componenti/accordion/#accordion-annidati